src/lib/layout/nav.svelte

```diff
@@ -12,6 +12,10 @@
 <nav>
 	<ul>
 		<li><a class={$page.path === '/' ? 'active' : ''} sveltekit:prefetch href="/">Home</a></li>
+		<li />
+		<li>
+			<a class={$page.path === '/about' ? 'active' : ''} sveltekit:prefetch href="/about">About</a>
+		</li>
```

Comment:
Can we use `class:active={$page.path == '/about'}` instead?

Reply:
> Can we use `class:active={$page.path == '/about'}` instead?

Sure, I can make that change but I'm not at my computer for awhile. Want to merge this and make an issue?

Reply:
updated here